packages/pwa/app/utils/routes-utils.js

```diff
@@ -15,11 +15,11 @@ import {urlPartPositions} from '../constants'
  * @param {object} - a custom configured object
  * @return {array} - list of configured route objects
  */
-export const configureRoutes = (routes = [], {ignoredRoutes = []}) => {
+export const configureRoutes = (routes = [], config, {ignoredRoutes = []}) => {
     if (!routes.length) return []
+    if (!config) return routes
 
-    const urlConfig = getUrlConfig()
-    if (!urlConfig) return routes
+    const {url: urlConfig} = config?.app
```

Comment:
I think we should throw an error here instead of having '?' optional chain. If config is not defined, the whole logic does not make sense anymore.

Reply:
ohh, I saw you have checked `if (!config) return routes` above, we might not need '?' in this sense then